uber/models.py

```diff
@@ -671,17 +671,17 @@ def search(self, text, *filters):
                 return attendees.filter(or_(*checks))
 
         def delete_from_group(self, attendee, group):
-            '''
+            """
             Sometimes we want to delete an attendee badge which is part of a group.  In most cases, we could just
             say "session.delete(attendee)" but sometimes we need to make sure that the attendee is ALSO removed
             from the "group.attendees" list before we commit, since the number of attendees in a group is used in
             our presave_adjustments() code to update the group price.  So anytime we delete an attendee in a group,
             we should use this method.
-            '''
+            """
```

Comment:
Style change; we generally use ``"""`` for docstrings.  Both ways work, but this is more consistent.